src/common/components/header/index.js

```diff
@@ -2,73 +2,99 @@ import React, { Component, PropTypes } from 'react';
 
 import { signOut } from '../../actions/auth-store';
 
-import containerStyles from '../../containers/container.css';
 import style from '../../style/vanilla/css/navigation.css';
+import {IconChevron, IconUser} from "../vanilla-modules/icons";
```

Comment:
There are couple of linting issues in this file (see Travis build log).

Reply:
Should be fixed now :)!